types/feather-icons/generated/feather-icons.d.ts

```diff
@@ -0,0 +1,5461 @@
+/** Declaration file generated by dts-gen */
+
+export const icons: {
+    activity: {
+        attrs: {
+            class: string;
+            fill: string;
+            height: number;
+            stroke: string;
+            "stroke-linecap": string;
+            "stroke-linejoin": string;
+            "stroke-width": number;
+            viewBox: string;
+            width: number;
+            xmlns: string;
+        };
+        contents: string;
+        name: string;
+        tags: string[];
+        toString: any;
+        toSvg: any;
```

Comment:
Generating the code like this is fine, but inefficient--both to read and to compile. You could save a lot by generating `type FeatherIconNames` here and fill in the repeated types in index.d.ts using a mapped type.

If the types aren't all  identical, you could still make things a lot more efficient by putting `type FeatherIcon` at the top and only using a different type for types that are actually different.